Sources/Target/Darwin/X86_64/ThreadX86_64.cpp

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2015, Facebook, Inc.
+// Copyright (c) 2015, Corentin Derbois <cderbois@gmail.com>
+// All rights reserved.
+//
+// This source code is licensed under the University of Illinois/NCSA Open
+// Source License found in the LICENSE file in the root directory of this
+// source tree. An additional grant of patent rights can be found in the
+// PATENTS file in the same directory.
+//
+
+#define __DS2_LOG_CLASS_NAME__ "Target::Thread"
+
+#include "DebugServer2/Target/Darwin/Thread.h"
+#include "DebugServer2/Architecture/CPUState.h"
+#include "DebugServer2/Host/Darwin/LibProc.h"
+#include "DebugServer2/Host/Darwin/PTrace.h"
+#include "DebugServer2/Target/Process.h"
+#include "DebugServer2/Utils/Log.h"
```

Comment:
`LibProc.h`, `PTrace.h`, `Log.h`, and `Process.h` do not seem to be required here.